src/stream/raspivid.js

```diff
@@ -2,20 +2,20 @@ const spawn = require('child_process').spawn;
 
 let streamProcess;
 
-function start() {
+function start(filename) {
   if (streamProcess) {
     return;
   }
 
-  streamProcess = spawn('raspivid', ['--segment 5000', '--wrap 5', '-t 0', '-o %d.h264']);
+  streamProcess = spawn('raspivid', ['--segment', 5000, '--wrap', 5, '-t', 0, '-o', filename]);
 }
 
 function stop() {
-  if (!streamProcess || !streamProcess.connected) {
```

Comment:
apparently, this was the problem. `.connected` means something else. Removing it give the right comportment